Documentation/operations/upgrade.rst

```diff
@@ -320,10 +320,18 @@ Helm Options
 * ``clustermesh.apiserver.tls.ca.cert`` has been deprecated in favor of ``tls.ca.key``, and will be removed in 1.13.
 * ``clustermesh.apiserver.tls.ca.key`` has been deprecated in favor of ``tls.ca.key``, and will be removed in 1.13.
 * ``tls.enabled`` has been removed as this attribute is not used at all.
+* ``tcFilterPriority`` has been added. See description of cilium's tc-filter-priority option in New Options, below.
 
 Only one CA will be generated with either the helm or CronJob auto method, there will be a short disruption while the new CA is
 propagated to all nodes.
 
+New Options
+~~~~~~~~~~~
+
+* ``tc-filter-priorty``: This option specifies the priority of the bpf filters installed by cilium. Previously, this was always '1'.
+This new option allows changing the priority so that other filters may be installed prior to the cilium filters, if needed. The
+default is '1' to maintain backward compatibility.
```

Comment:
- Case on BPF, Cilium
- RST syntax on \`\`1\`\`
- Indentation to mark the paragraph as a list item
- Wrapping

```suggestion
* ``tc-filter-priorty``: This option specifies the priority of the BPF filters
  installed by Cilium. Previously, this was always ``1``. This new option
  allows changing the priority so that other filters may be installed prior to
  the Cilium filters, if needed. The default is ``1`` to maintain backward
  compatibility.
```